prow/scripts/cluster-integration/kyma-gke-long-lasting.sh

```diff
@@ -271,6 +279,25 @@ function addGithubDexConnector() {
     go run "${KYMA_PROJECT_DIR}/test-infra/development/tools/cmd/enablegithubauth/main.go"
 }
 
+function applyServiceCatalogCRDOverride(){
+    shout "Apply override for ServiceCatalog"
```

Comment:
```suggestion
    shout "Apply override for ServiceCatalog to enable CRD implementation"
```

the function name is verbose enough but the comment printed in console does not say too much